Palace/EkirjastoDigitalMagazine/DigitalMagazineBrowserViewController.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  EkirjastoEpaperViewController.swift
+//  Ekirjasto
+//
+//  Created by Johannes Ylönen on 13.3.2024.
+//  Copyright © 2024 The Palace Project. All rights reserved.
```

Comment:
This would be part of E-kirjasto project, which derived from the Palace Project to meet Finnish Public Libraries' requirements. The Readme file has to be updated.